docs/MapColonies/Raster/services/polygon_parts.md

```diff
@@ -0,0 +1,877 @@
+---
+id: raster-polygon-parts-service
+slug: polygon-parts
+title: Polygon Parts 🧩
+description: Search, Filter and Aggregate polygon parts (vectorial) layers
+tags:
+  - wfs
+  - raster
+  - features
+  - API
+  - polygon-parts
+---
+
+# Polygon Parts Service 🧩
+
+The raster polygon parts service uses the [WFS](/docs/ogc/protocols/ogc-wfs) protocol which is a READ-ONLY Web Feature Service.
+it provides facilities for searching and retrieving feature data with the `GetCapabilities`, `DescribeFeatureType` and `GetFeature` operations all other operations are invalid.
+
+For the full capabilities provided by the service see [GetCapabilities](/docs/ogc/protocols/ogc-wfs#getcapabilities)
+
+WFS partitions the queryable data into feature types each feature type has its own attributes describing the structure of the data itself.
+For the description of a feature type see [DescribeFeatureType](/docs/ogc/protocols/ogc-wfs#describefeaturetype)
+
+Finally, Features can be retrieved with the [GetFeature](/docs/ogc/protocols/ogc-wfs#getfeature) operation. the request returns a collection of geographic feature instances filtered according to a criteria set by the requesting client.
+<br/>
+
+:::important
+**In polygon parts, each feature represent single polygon**
+:::
+
+:::caution
+**Authentication must be integrated in order to communicate with Map and Catalog services.**<br/>
+**See the principles [here](/docs/MapColonies/authentication)**
+:::
+
+## Example
+
+WFS protocol partitions the queryable features into different feature types.
+Each feature types actually represent other catalog layer [Raster Catalog](/docs/MapColonies/Raster/services/catalog/raster-catalog-profile-v1).
+
+:::info
+Each polygon parts data managed on individual db tables per each.layer
```

Comment:
why do the user needs to know how we store the data?

Reply:
it explain the purpose that each feature type represent single layer, that stored on dedicated table